src/nhc2-platform.ts

```diff
@@ -121,31 +122,72 @@ class NHC2Platform implements DynamicPlatformPlugin {
       },
       generic: {
         service: this.Service.Switch,
-        handlers: [this.addTriggerCharacteristic],
+        handlers: [this.addTriggerCharacteristic]
       },
-      sunblind : {
-        service : this.Service.WindowCovering,
-        handlers : [this.addPositionChangeCharacteristic]
+      'switched-generic': {
+        service: this.Service.Switch,
+        handlers: [this.addStatusChangeCharacteristic],
```

Comment:
Sometimes I see trailing commas, and sometimes I don't. Could you make a change to make it consistent?

Reply:
Fixed